docs/nodes/light-node.md

```diff
@@ -1,32 +1,32 @@
 # Setting Up A Celestia Light Node
 
-This tutorial will guide you through setting up a Celestia Light Node,
-which can allow you to do data-sampling on the Data Availability (DA)
-network.
+This tutorial will guide you through setting up a Celestia light node, which
+will allow you to perform data availability sampling on the data
+availability (DA) network.
 
-> To view a video tutorial for setting up a Celestia Light Node, click [here](../developers/light-node-video.md)
+> To view a video tutorial for setting up a Celestia light node, click [here](../developers/light-node-video.md)
 
 ## Overview of Light Nodes
 
-Light nodes (CLN) ensure data availability. This is the most common
+Light nodes ensure data availability. This is the most common
 way to interact with the Celestia network.
 
-> Note: In future implementations, Light Nodes can also publish
-  transactions ([see ADR](https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-004-state-interaction.md)),
-  though in Mamaki, transactions are left to Bridge Nodes.
+> Note: In future implementations, light nodes will be able to publish
+  transactions ([see ADR](https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-004-state-interaction.md)).
+  However in Mamaki, transactions can only be submitted by Bridge Nodes.
```

Comment:
Hi @renaynay @Wondertan can light nodes now submit transactions? I believe the API now allows it, right?

Reply:
You're right @YazzyYaz ! I was able to submit a PFD on a light node with:

```bash
[ec2-user@ip-172-31-15-92 celestia-node]$ curl -X POST -d '{"namespace_id": "c14da9d459dc57f5", "data": "4f7a3f1aadd83255b8410fef4860c0cd2eba82e24a", "gas_limit": 60000}'  localhost:26658/submit_pfd
```

https://celestiascan.vercel.app/transaction/B9F085F62151FC6CDD3A180B4BF8437602DE088C5C993376D3737FD6871335B6

so I'll remove this `Note:`